source/jwtd/jwt.d

```diff
@@ -233,13 +233,32 @@ SQIDAQAB
 -----END PUBLIC KEY-----
 EOS";
 
-string es256_key = q"EOS
+	version(UseBotan) {
+		string es256_key = q"EOS
+-----BEGIN PRIVATE KEY-----
+MIGEAgEAMBAGByqGSM49AgEGBSuBBAAKBG0wawIBAQQgHxxA+0sQXmE4myibmhVT
+l0ymANRHZBi4lNd22/F7NCWhRANCAAQy5KexZuIg/J8UAgC+VuWI85SdCWJtvrvI
+TolSpdVp69vxmisrYd/F8WD2kZWGDdIa4EJsdwnzhYo5fcZIwTBw
+-----END PRIVATE KEY-----
+EOS";
+	}
+	else {
+		string es256_key = q"EOS
 -----BEGIN EC PRIVATE KEY-----
 MHQCAQEEIB8cQPtLEF5hOJsom5oVU5dMpgDUR2QYuJTXdtvxezQloAcGBSuBBAAK
 oUQDQgAEMuSnsWbiIPyfFAIAvlbliPOUnQlibb67yE6JUqXVaevb8ZorK2HfxfFg
 9pGVhg3SGuBCbHcJ84WKOX3GSMEwcA==
 -----END EC PRIVATE KEY-----
-EOS"; 
+EOS";
+	}
+
+	string es256_pubkey = q"EOS
```

Comment:
This I added to verify with Botan. Again generated from the above PK.
But should not this be used also with OpenSSL? Now it verifies with the private key.
Because of that I left it outside the version block